docs/en/faq.md

```diff
@@ -179,28 +151,12 @@ We list some common troubles faced by many users and their corresponding solutio
 - "RuntimeError: Expected to have finished reduction in the prior iteration before starting a new one"
 
   1. This error indicates that your module has parameters that were not used in producing loss. This phenomenon may be caused by running different branches in your code in DDP mode.
-  2. You can set `find_unused_parameters = True` in the config to solve the above problems(but this will slow down the training speed.
-  3. If the version of your MMCV >= 1.4.1, you can get the name of those unused parameters with `detect_anomalous_params=True` in `optimizer_config` of config.
+  2. You can set `find_unused_parameters = True` in the config to solve the above problems, but this will slow down the training speed.
+  3. You can get the name of those unused parameters with `detect_anomalous_params = True`. Note `detect_anomalous_params = True` will slow down the training speed, so it is recommended for debugging only.
```

Comment:
Will this usage be supported in mmengine?

Reply:
line 155 is a bit vague, we should tell users where to add it.
It has been in model_wrapper_cfg now
https://github.com/open-mmlab/mmengine/blob/main/mmengine/model/wrappers/distributed.py#L91

Reply:
I think it should be add in configs or model_wrapper_cfg

Reply:
I think it should be add in configs or model_wrapper_cfg

Reply:
I think it should be add in configs or model_wrapper_cfg